adoc/Release-Notes.adoc

```diff
@@ -33,6 +33,174 @@ endif::[]
 [id='sec.major-change']
 == Major Changes
 
+[id='sec.1_5']
+=== Release 1.5, September 2019
+
+[id='sec.1_5.new']
+==== What Is New?
+
+* {scf} has been updated to version 2.18.0:
+** PXC (Percona XtraDB Cluster) replaces `cf-mysql` for database management -- please read the Known Issues section for this version on deployment and upgrade changes
+** Ability to set `config.HA_strict=false` in combination with `config.HA=true` to allow lowering the sizing count for a role below what is required for HA
+** UAA can now be embedded in the `cf` namespace -- please read the Known Issues section for this version on deployment limitations
+* The Stratos UI has been updated to version 2.5.1:
+** Tech preview for helm feature
+** Add custom welcome message on endpoints page
+** Add support for connecting {caasp} V4 endpoints
+** Refinements to the Autoscaler UI
+** For a full list of features and fixes see https://github.com/SUSE/stratos/releases/tag/2.5.1.
+
+For information about deploying and administering {product}, see the product manuals at
+{doc-url}.
+
+[id='sec.1_5.feature']
+==== Features and Fixes
+* `cf-deployment` has been updated to version 9.5.0.
+* Eirini updated to 0.0.14
```

Comment:
"Eirini tech preview now ships Eirini version 0.0.14"

Reply:
We're using existing release note wording here.